smac/utils/io/cmd_reader.py

```diff
@@ -3,7 +3,8 @@
 import re
 from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter, SUPPRESS
 
-from smac.optimizer.ei_optimization import ChooserNoCoolDown, ChooserLinearCoolDown
+from smac.optimizer.random_configuration_chooser import ChooserNoCoolDown, \
```

Comment:
I can't add a comment below, but now the default value for RandomChooser needs to be changed

Reply:
The default values for the choosers were changed in all places (modulus = 1.0 changed to modulus = 2.0).

Reply:
Sorry, I meant in the docstring :-)